### ISSUE TITLE
Labels for other frames in sequence

### ISSUE THREAD

**Issue by deepakraina99** [mattpfr/lmanet#10]
In the paper, you mention that "The training and validation sets provide dense ground truth annotations, but only for the 20th
frame of each sequence. We therefore train and evaluate our approach on video snippets but report mean Intersection over
Union (mIoU) results on the provided 20th frame"

Do you mean the the gorund truth labels for all sequences in the interval during training is 20th frame?

Is this a fair assumption?
